js-client/packages/pipe/src/lib/pipe.ts

```diff
@@ -42,7 +43,7 @@ export class Pipe {
     }
 
     topic<TNotifications extends Record<string, unknown>>(topicType: string, topic: unknown) {
-        let subscription = find(this.#subscriptions, { topic, topicType })
+        let subscription = this.#subscriptions.find(t => deepEqual(t.topic, topic) && t.topicType === topicType)
```

Comment:
We could extract this comparison to separate function and keep it point free like `matches` previously